src/components/map/LocationMarker.jsx

```diff
@@ -37,7 +37,7 @@ class LocationMarker extends Component {
   }
 
   renderUrl(url) {
-    const linkUrl = url.includes('//') ? url : `http://${url}`;
+    const linkUrl = url.indexOf('//') > -1 ? url : `http://${url}`;
```

Comment:
Any reason for preferring "indexOf" over "includes"?
My thinking is usually that the most specific function that serves the purpose is the one to use (e.g. preferring "startsWith" even though "indexOf === 0" could achieve the same).

Reply:
`includes` is not supported on Chrome on Android 5.1, and would throw an error on clicking the marker. This is mentioned on row 4 of the bugs list.